website/data/users.yml

```diff
@@ -23,10 +23,10 @@
   greyImage: /images/users/used_by_babel.svg
   infoLink: https://babeljs.io
   pinned: true
-- caption: Webpack CLI
+- caption: Webpack
```

Comment:
I’m pretty sure it’s “webpack” (lowercase w).

Reply:
https://github.com/prettier/prettier/commit/79196030a78954359cb7885ba32b984e0e960510